bootstrap/mac.sh

```diff
@@ -6,3 +6,13 @@ fi
 
 brew install augeas
 brew install dialog
+
+if ! hash pip 2>/dev/null; then
+    echo "pip Not Installed\nInstalling python from Homebrew..."
+    brew install python
```

Comment:
`virtualenv` will generate its own localized versions of `pip`. I solely used pip with virtualenv currently.